sklearn/linear_model/tests/test_logistic.py

```diff
@@ -570,6 +570,53 @@ def test_logistic_regressioncv_class_weights():
     assert_array_almost_equal(clf_lib.coef_, clf_sag.coef_, decimal=4)
 
 
+def test_logistic_regression_sample_weights():
+    X, y = make_classification(n_samples=20, n_features=5, n_informative=3,
+                               n_classes=3, random_state=0)
```

Comment:
I am not sure that `n_classes=3` brings something interesting in this test.
`n_classes=2` would reduce the run-time which is still quite long (0.76 to 0.29 sec).